nixos/modules/installer/cd-dvd/installation-cd-graphical-gnome.nix

```diff
@@ -33,46 +26,17 @@ with lib;
     };
   };
 
-  environment.systemPackages =
-    [ # Include gparted for partitioning disks.
-      pkgs.gparted
-
-      # Include some editors.
-      pkgs.vim
-      pkgs.bvi # binary editor
-      pkgs.joe
-
-      pkgs.glxinfo
-    ];
-
-  # Don't start the X server by default.
-  services.xserver.autorun = mkForce false;
-
-  # Auto-login as root.
+  # Auto-login as nix-live.
   services.xserver.displayManager.gdm.autoLogin = {
     enable = true;
-    user = "root";
+    user = "nix-live";
   };
 
-  system.activationScripts.installerDesktop = let
-    # Must be executable
-    desktopFile = pkgs.writeScript "nixos-manual.desktop" ''
-      [Desktop Entry]
-      Version=1.0
-      Type=Link
-      Name=NixOS Manual
-      URL=${config.system.build.manual.manual}/share/doc/nixos/index.html
-      Icon=system-help
-    '';
+  system.activationScripts.installerGnome = ''
+    mkdir -p /home/nix-live/Desktop
+    chown nix-live /home/nix-live /home/nix-live/Desktop
 
-  # use cp and chmod +x, we must be sure the apps are in the nix store though
-  in ''
-    mkdir -p /root/Desktop
-    ln -sfT ${desktopFile} /root/Desktop/nixos-manual.desktop
-    cp ${pkgs.gnome3.gnome-terminal}/share/applications/gnome-terminal.desktop /root/Desktop/gnome-terminal.desktop
-    chmod a+rx /root/Desktop/gnome-terminal.desktop
-    cp ${pkgs.gparted}/share/applications/gparted.desktop /root/Desktop/gparted.desktop
-    chmod a+rx /root/Desktop/gparted.desktop
+    ln -sfT ${pkgs.gnome3.gnome-terminal}/share/applications/gnome-terminal.desktop /home/nix-live/Desktop/gnome-terminal.desktop
```

Comment:
GNOME does not have desktop by default so I guess this is moot. You would need something like:

```nix
services.xserver.desktopManager.gnome3 = {
  extraGSettingsOverridePackages = with pkgs; [ gnome3.gnome_shell ];
  extraGSettingsOverrides = ''
    [org/gnome/shell]
    favorite-apps=['firefox.desktop', 'nixos-manual.desktop', 'gparted.desktop', 'org.gnome.Terminal.desktop']
  '';
};
```

Reply:
Ok. My solution for now is to remove gnome desktop entries and let someone more familiar with gnome take care of improving that installer.